### PR TITLE
display name instead of holder addr in collection

### PR DESCRIPTION
**Pull request by 0xaptosj**
Screenshot 2023-10-20 at 11 57 40 AM

### CODE REVIEW

File: frontend/src/app/home/Pet/Collection.tsx [hunk 1]
```diff
@@ -98,15 +107,11 @@ export function Collection() {
       <label htmlFor="owner_field">
         Aptogotchi Minted: {collection?.current_supply}
       </label>
-      <label htmlFor="owner_field">All Holders</label>
+      <label htmlFor="owner_field">Fellow aptogotchis</label>
```
Comment (0xaptosj) [inline frontend/src/app/home/Pet/Collection.tsx:110]: what's the proper wording here? first 3 aptogotchis?

Comment (angieyth) [reply frontend/src/app/home/Pet/Collection.tsx:110]: how about adding **_... and more_** right after those three names? and remove the `[ ]` around the names?

Comment (0xaptosj) [reply frontend/src/app/home/Pet/Collection.tsx]: updated

**Review by angieyth — APPROVED** [review]
love love love

**Pull request merged**